client/state/data-layer/wpcom/activity-log/rewind/to/index.js

```diff
@@ -34,8 +35,13 @@ const requestRestore = action =>
 		action
 	);
 
-export const receiveRestoreSuccess = ( { siteId, timestamp }, restoreId ) =>
-	getRewindRestoreProgress( siteId, restoreId );
+export const receiveRestoreSuccess = ( { siteId, timestamp }, restoreId ) => [
+	getRewindRestoreProgress( siteId, restoreId ),
+
+	// once we start Rewinding we want to ensure that we
+	// start tracking it and updating the "state machine"
```

Comment:
is it a state machine or is it a "state machine" ? 🙃

Reply:
since many APIs and processes are modeled as a state machine I have never seen it fit to claim that name exclusively here, so within the Jetpack Rewind world the "state machine" API refers explicitly to `/sites/{ site }/rewind` as well as the state machine it represents. therefore the quotes are mainly meant to refer not to the fact that it's a state machine but rather to the specific API which is called by that name.

not sure if it's worth changing here since it's used all over the place. probably worth a string review after the code is settled.